source/server/hot_restart_impl.cc

```diff
@@ -101,7 +101,9 @@ std::string SharedMemory::version(size_t max_num_stats, size_t max_stat_name_len
                      max_stat_name_len);
 }
 
-std::string SharedMemory::version() { return version(num_stats_, Stats::RawStatData::size()); }
+std::string SharedMemory::version() {
+  return version(num_stats_, Stats::RawStatData::maxNameLength());
```

Comment:
Sorry I think I'm confused. How does this matter? What's the difference if it uses the full size?

Reply:
Or is it just mismatched from what the other thing is returning?

Reply:
I see, never mind.